docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -762,11 +762,13 @@ Running the preceding method with a value of `15` leads to setting a tag with a
 
 //=== Spring Integration
 
-// TODO: Update this
-
-// === HTTP
+=== HTTP
 
-// TODO: Update this
+If a customization of client / server parsing of the HTTP related spans is required,
```

Comment:
I might say what you have and link to brave's docs for more

ex https://github.com/openzipkin/brave/tree/master/instrumentation/http#sampling-policy shows how to make a path-based sampler